setup.py

```diff
@@ -43,7 +43,10 @@ def py2_only(*dependencies):
         "Programming Language :: Python :: 3.6",
         "Topic :: Software Development",
     ],
-    install_requires=["graphviz", "matplotlib"],
+    install_requires=[],
+    extras_require=[
```

Comment:
This should be a dict; that why [Travis failed](https://travis-ci.org/jacquev6/ActionTree/jobs/283680088#L616)